tests/docker-load-test.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# NOTE: This script should only run inside a Node.js docker container.
+# See ./load-test.sh for details.
+set -ex
+
+# Create an empty directory to test
+mkdir -p /app && cd /app
+tar zxf /pulsar-client-node/pulsar-client-node.tar.gz
+cp /pulsar-client-node/build/Release/Pulsar.node .
```

Comment:
You can just copy `Pulsar.node` to `/pulsar-client-node/lib/binding`

In this way, does not need to change its: `require()`

Reply:
No. If we're doing this way, we need to run `npm install` to install these dependencies: https://github.com/apache/pulsar-client-node/blob/dc2c3c5feb1d51e317215ae183ccf0a1b7306353/package.json#L55-L57

Actually, IMO they are not required. I'm not sure if there is a way to move them from `dependencies` to `devDependencies`. But this script is just for tests and they are equivalent.

Reply:
> If we're doing this way, we need to run npm install to install these dependencies

This is ok, this is consistent with how users use.

> And npm install might build the Pulsar.node again on the test docker image, so the Pulsar.node built on CentOS 7 might be overwritten.

When `/pulsar-client-node/lib/binding/Pulsar.node`, It will no longer be compiled and overwritten.

Reply:
> When /pulsar-client-node/lib/binding/Pulsar.node, 

Did you mean when `/pulsar-client-node/lib/binding/Pulsar.node` already exists?